build.cmd

```diff
@@ -4,51 +4,96 @@ PUSHD %~dp0
 SETLOCAL
 SETLOCAL ENABLEDELAYEDEXPANSION
 
+:Loop
+if [%1]==[] GOTO Begin
+
+if /I [%1]==[Debug] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[Release] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[PROD] (
+    SET Environment=PROD
+    GOTO Next
+)
+
+if /I [%1]==[raw] (
+    SET UpdateTemplate=%1
```

Comment:
build template => only update template, not build project

build raw => not update template, only build project

---
In reply to: [68713739](https://github.com/dotnet/docfx/pull/435#discussion_r68713739) [](ancestors = 68713739)